addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/internal/data/Thermostat.java

```diff
@@ -183,11 +183,15 @@ public void setFanTimerDuration(Integer duration) {
         fanTimerDuration = duration;
     }
 
+    public Integer getTimeToTarget() {
+        return parseTimeToTarget(timeToTarget);
+    }
+
     /*
      * Turns the time to target string into a real value.
      */
-    public Integer getTimeToTarget() {
-        if (timeToTarget.startsWith("~")) {
+    static Integer parseTimeToTarget(String timeToTarget) {
+        if (timeToTarget.startsWith("~") || timeToTarget.startsWith("<")) {
             timeToTarget = timeToTarget.substring(1);
```

Comment:
Please do not reassign method parameters

Reply:
And thanks for your pr 😉

Reply:
Fixed!